plugins/enter.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: enter
+spec:
+  version: "v1.0.0"
+  homepage: https://github.com/kvaps/kubectl-enter
+  shortDescription: Exec into node via kubectl 
+  description: |
+    Tiny plugin for getting shell into the node.
```

Comment:
I think the mechanics of your plugin are simple enough to mention here. Something like

> Start a root shell in the node's host OS running. You need to be able to start privileged containers for that.